README.md

```diff
@@ -117,9 +126,13 @@ https://user-images.githubusercontent.com/79102959/134894112-e4f38ced-0992-428c-
 |   name*	|  Country component name  	|  string 	|  - 	|
 |   type*   | Must be `country` | string | - |
 |   label	|  Text shown over the coutnry list|  string  	|   ''	|
+|   language	| An string with the language shortcode in which you want to display the country names opt:  <br /> Available laguages: `es`,`de`,`fr`,`en`  	| string   	|   en	|
 |   placeholder    |   Placeholder displayed in the select    |    string       |   ''   |
 |   priorityOptions    |   Array of strings with shortcode(s) of the countries that want to be displayed first in the countries list. Ex: ['GB', 'ES']    |    string       |   '' |
 |   returnCountryName	|  Returns the country name (in English) as each option value, instead of the country code  	|  boolean 	|  false	|
+|   **countryAndRegionsData**	| An object or array of objects with the acronym(s) and the names of the countries that you want to be shown in the select.  	| json   	|  [] 	|
+|  countryName     |   The name of the country |  string     | ''
+|  countryShortCode     |   The name of the country |  string     | ''
```

Comment:
Probably a better description should be added here.` Country short code ` for instance